README.md

```diff
@@ -22,9 +22,9 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
     TiDB acts like it is a MySQL 5.7 server to your applications. You can continue to use all of the existing MySQL client libraries, and in many cases, you will not need to change a single line of code in your application. Because TiDB is built from scratch, not a MySQL fork, please check out the list of [known compatibility differences](https://docs.pingcap.com/tidb/stable/mysql-compatibility).
 
-- __Distributed Transactions with Strong Consistency__
+- __Distributed Transactions__
```

Comment:
LGTM for this change because I've checked the [definition of 'consistency' on Wikipedia](https://en.wikipedia.org/wiki/Consistency_(database_systems)#As_an_ACID_guarantee):

> The guarantee that database constraints are not violated, particularly once a transaction commits.

So it is hard to say 'strong consistency' for TiDB(in the context of 'transaction') since we accept `FOREIGN KEY` keywords but ignore the constraint.